scripts/csv_to_geojson.py

```diff
@@ -0,0 +1,93 @@
+"""
+Simply script to convert a csv file of prediction results into a geojson
+
+Specify the CSV filepath and your confidence threshold. The CSV file should
+contain tile indices in TMS coordinates. The confidence threshold is useful if
+you ran prediction over a large area -- it will keep the resulting geojson
+much smaller on disk. Set the threshold to 0.0 to include all tile predictions
+in the geojson output.
+
+Requires pygeotile, geojson, and mercantile
+"""
+
+import csv
+import json
+import argparse
+import os.path as op
+
+from mercantile import feature, Tile
+from geojson import Feature
+from pygeotile.tile import Tile as Pygeo_tile
+
+
+def convert_csv(fname_csv, fname_geojson, tile_format, thresh_ind, thresh):
+    """Convert tile indices in CSV file to geojson"""
+
+    if not op.exists(fname_csv):
+        raise ValueError(f'Cannot find file {fname_csv}')
+
+    # Error check tile format
+    if tile_format == 'tms':
+        tile_func = Pygeo_tile.from_tms
+    elif tile_format == 'google':
+        tile_func = Pygeo_tile.from_google
+    else:
+        raise ValueError(f'Tile format not understood. Got: {tile_format}')
+
+    if not 0 <= thresh <= 1.:
+        raise ValueError(f"'thresh' must be on interval [0, 1]. Got: {thresh}")
+
+    with open(fname_csv, 'r') as csvfile:
+        with open(fname_geojson, 'w') as results:
+            reader = csv.reader(csvfile)
+            first_line = True
+
+            # Create a FeatureCollection
+            results.write('{"type":"FeatureCollection","features":[')
+            next(reader)  # Skip header
+
+            for row in reader:
+
+                # Load as pygeotile using TMS coords
+                geot = tile_func(*[int(t) for t in row[0].split('-')])
+
+                # Create feature with mercantile
+                feat = feature(Tile(geot.google[0], geot.google[1], geot.zoom))
```

Comment:
@wronk can you correct this line to use TMS (which is what mercantile expects) rather than google?

Reply:
I think mercantile expects X/Y/Z (google) format

https://mercantile.readthedocs.io/en/latest/api/mercantile.html#mercantile.feature

Reply:
XYZ isn't google; I just ran it on a large data set and it put all my tiles in the ocean

Reply:
...because XYZ is google and I was using the wrong input type. That's my bad, this is good to merge as is